docker-compose.yml

```
@@ -41,7 +41,7 @@ services:
       - physical-address-annotator
 
   phi-deidentifier-app:
-    image: nlpsandbox/phi-deidentifier-app:0.3.2
+    image: nlpsandbox/phi-deidentifier-app:1.0.0
```

The latest version of the PHI Deidentifier is 1.0.1

```suggestion
    image: nlpsandbox/phi-deidentifier-app:1.0.1
```

---

@tschaffter the name of this branch, as per [your instructions](https://github.com/nlpsandbox/nlpsandbox-schemas/issues/179), is `release-1.0.0`. I don't think we ever released a branch `1.0.0`.